.github/workflows/analysis.yml

```diff
@@ -16,6 +16,51 @@ concurrency:
 jobs:
   #TODO add the Unit TEST with sonar, sample https://github.com/bcgov/quickstart-openshift/blob/main/.github/workflows/analysis.yml#L17-L58
 
+  tests:
+    name: Unit Tests
+    # TODO - Re-enable below before submitting PR, jsut have it disabled while I work via draft.
+    # if: ${{ ! github.event.pull_request.draft }}
+    runs-on: ubuntu-24.04
+    timeout-minutes: 5
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    strategy:
+      matrix:
+        dir: [backend/sites, frontend/site-registry]
+        include:
+          - dir: backend
+            token: SONAR_TOKEN_BACKEND
+          - dir: frontend
+            token: SONAR_TOKEN_FRONTEND
+    steps:
+      - uses: bcgov-nr/action-test-and-analyse@v1.2.1
+        with:
+          commands: |
+            npm ci
+            npm run test:cov
+          dir: ${{ matrix.dir }}
+          node_version: "22"
+          sonar_args: >
+            -Dsonar.exclusions=**/coverage/**,**/node_modules/**,**/*spec.ts
+            -Dsonar.organization=bcgov-sonarcloud
+            -Dsonar.projectKey=quickstart-openshift_${{ matrix.dir }}
```

Comment:
this needs to be `nr-site-registry-${{ matrix.dir }}` from `quickstart-openshift_${{ matrix.dir }}`

Reply:
Done, however I named with dashes instead of underscore for consistency elsewhere.  Let me know if that's incorrect.